draft-ietf-quic-ack-frequency.md

```diff
@@ -498,19 +498,20 @@ can delay a sender's response to network congestion or cause it to underutilize
 the available bandwidth.
 
 To limit the consequences of reduced acknowledgment frequency, a sender
-can cause a receiver to send an acknowledgment at least once per round trip
-when there are ack-eliciting packets in flight.
+can utilize the extension in this draft to request a receiver
+to send an acknowledgment at least once per round trip,
+when there are ack-eliciting packets in flight, the following ways:
```

Comment:
Does the colon mean that a list follows?

Reply:
I don't think it has to follow a list...

Reply:
It seems weird to me to use the colon this way, but also is there a word missing, like "in the following ways"?